tests/x11/gnome_tweak_tool.pm

```diff
@@ -15,8 +15,15 @@ use strict;
 use testapi;
 
 sub run {
+    my @gnome_tweak_matches = qw(gnome-tweaks gnome-tweak-tool command-not-found);
+
     mouse_hide(1);
-    x11_start_program('gnome-tweak-tool');
+    x11_start_program('gnome-tweaks', target_match => \@gnome_tweak_matches);
+    if (match_has_tag('command-not-cound')) {
```

Comment:
TBH the problem is *here*, a typo `cound` , thus even the needle has matched it still not going to try another command